docs/create_usb_boot_disks.rst

```diff
@@ -0,0 +1,214 @@
+Create USB Boot Disks
+=====================
+
+Overview
+-------------
+For the initial SecureDrop setup, you will need three USB disks.
+
+Two of them will will be used for the Tails operating system,
+for creating the *Admin Workstation* and *Secure Viewing Station*.
+
+The other USB disk will have the Ubuntu Server installer,
+which is needed to install the underlying server OS for your
+*application* and *monitor* servers.
```

Comment:
Nit: capitalize and italicize the entire term (i.e. *Application* and *Monitor Servers*).